### DISCUSSION TITLE
Building a Web-Based Video Editor: Seeking Advice for Development

### DISCUSSION THREAD

**Discussion by adityaasir** [community/community#126287] [Programming Help]
### Select Topic Area

General

### Body

I'm a web developer embarking on a long-term project to create a web-based application similar to Alight Motion. I'm reaching out to the community to gain some insights on the initial steps involved in bringing this project to fruition.

In addition to the development process, I'm also curious about how to list my app on various websites like Modyedge. Any guidance on the listing process would be greatly appreciated.

I'm eager to learn from experienced developers and collaborate with the GitHub community as this project progresses. Thank you for your time and support! You can check more about listing [here](https://modyedge.com/alight-motion-mod-apk/), to provide a detailed guide on it.

**Comment by nangerno** [comment]
Hello, I already have some experience with this field.
I used Qlips API to integrate video editing functions to my platform.
You can find more detailed information about "Qlips" as AI video editing API online. 
I can help you for this approach.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @adityaasir ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by adasarpan404** [comment]
To build the web-based video editor, you have to understand the FFMPEG package.
[FFMPEG](https://ffmpeg.org/).
